src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -804,11 +809,14 @@ function prepareStruct(struct) {
   }
   if ('item' in struct) {
     ret.items = []
-    struct.item.forEach((item, index) => {
+    let lastFieldId = -1;
```

Comment:
Not a big deal, but for some or the other reason, your editor is not picking the code formatting. We have at some point mandated the no-semicolons and at least in my case VSCode and Prettier plugins with husky git hooks are reformatting this automatically all the time. I am not sure how you avoided that....

But like I said: not a big deal. These gets weeded out sooner or later when someone saves these files in their editors.

Reply:
Fwiw, the "no semicolons" format is full of terrible footguns, so I'm surprised you are using it...

Also, clearly these things are not weeded out in general: tons of files have things that husky catches as formatting errors if I run it.  It might be worth doing a single mass-reformat to fix them all.